spec/services/halls/update_spec.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Halls::Update do
+  let!(:hall) { create :hall, name: 'name', capacity: 40 }
+  let(:params) do
+    {
+      name:,
+      capacity: 41
+    }
+  end
+  let(:name) { 'name updated' }
```

Comment:
```suggestion
  let(:name) {  Faker::Name.name }
```
I recommend using faker, here you have some available [methods](https://github.com/faker-ruby/faker#default)